### PR TITLE
Update composer version in the documentation

### PR DESCRIPTION
**Pull request by Remiii**
Update composer version in the documentation because some options in the doc need to be with the development version (exemple: customize the default button look).

### CODE REVIEW

**Comment by craue** [comment]
I'd rather keep the instructions like that to not encourage people to use an unstable version by default.

**Comment by Remiii** [comment]
@craue Ok I understand, but some options are not available in the current stable version. So we need to precise them. What do you think?

**Comment by craue** [comment]
How about just removing the version number and letting Composer choose which version to install depending on the `minimum-stability` setting. I even more like this approach because I won't have to remember updating this value on each major release. :smile:

File: README.md [hunk 1]
```diff
@@ -20,14 +20,12 @@ A live demo showcasing these features is available at http://craue.de/sf2playgro
...
 
-Let Composer download and install the bundle by running
+Let Composer download and install the bundle by running in a shell.
 
 ```sh
-php composer.phar require craue/formflow-bundle:~2.0
+php composer.phar require craue/formflow-bundle:dev-master
 ```
 
-in a shell.
```
Comment (craue) [inline README.md]: Why do you want to move this part?

Comment (Remiii) [inline README.md]: For me it is more explicit in one sentence, but it's not important.

**Comment by Remiii** [comment]
@craue Sure It's better to remove the version number and letting Composer choose alone. But we need to precise the minimum version for all options not available in the stable version (with something like this: `option available in master branch and not in the development branch`). Ok for you?

File: README.md (hunk 1 above)
Comment (craue) [inline README.md]: For me it is a sentence in its original form. The code could be inlined to make it look better, but it's like that in all my bundles currently and I'd like to keep it consistent.

**Comment by craue** [comment]
Considering [this explanation](https://github.com/KnpLabs/KnpMenuBundle/pull/257#discussion-diff-23600236) I now think it's better to keep the version number to explicitly install the version someone is reading the docs for. So I'm basically :+1: for your initial suggestion, although I'd prefer `~3.0@dev`.

This would also solve the issue of feature confusion as someone is reading the docs for a specific version, right?

**Comment by craue** [comment]
@Remiii: Are you still working on this?

**Comment by Remiii** [comment]
@craue: I will look that ASAP. Sorry for the delay.

**Comment by Remiii** [comment]
@craue: You got it :sunglasses:

**Comment by craue** [comment]
Really? Now the change for the version number is gone. :scream:
Having `~3.0@dev` there would be fine.

Do you still think mentioning in the docs for the master branch that a feature from the master branch is not available in the latest stable version? If someone is using a specific version, he/she should also read the docs for that version, not for the master branch. So I don't see the need to add such notices.

**Comment by Remiii** [comment]
@craue: Ok but a warning like `feature not available in all version` is better than nothing? What do you think?

**Comment by craue** [comment]
I don't see that it's needed at all. When reading the docs for a specific version, you can only read about all the features actually available, not stumble upon the ones added in later versions. But notices like these would have to be removed again when releasing a new version, which might be forgotten.

**Comment by Remiii** [comment]
@craue: If you think is not a good thing you can close the PR.

**Comment by craue** [comment]
I thought the main purpose of this PR is to update the version number for the Composer command.

**Pull request closed without merging**